src/main/resources/db/migration/R__TypeCodes.sql

```diff
@@ -76,7 +76,10 @@ VALUES (1, 'Tree'),
        (2, 'Shrub'),
        (3, 'Forb'),
        (4, 'Graminoid'),
-       (5, 'Fern')
+       (5, 'Fern'),
+       (6, 'Fungi'),
```

Comment:
Are we sure we want "Fungi" instead of "Fungus?" All the other options are singular nouns.

Reply:
Good question, I will check with Allison. Thanks!!